api/models/User.js

```diff
@@ -1,15 +1,19 @@
 var bcrypt = require('bcrypt');
```

Comment:
This file should be changed to something like:

`module.exports = parseFloat(sails.version) < 1 ? require('../lib/user.v1') : require('../lib/user.v2');`